tests/e2e/browser/アクセント分割.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from "@playwright/test";
+
+import { navigateToMain } from "../navigators";
+
+test.beforeEach(async ({ page }) => {
+  const BASE_URL = "http://localhost:5173/#/home";
+  await page.setViewportSize({ width: 800, height: 600 });
+  await page.goto(BASE_URL);
+});
+
+test("アクセント分割したらアクセント区画が増える", async ({ page }) => {
```

Comment:
すいませんこちらのコメントが間違っていて、区画じゃなくて区間でした 🙇 
勝手ながら修正させていただきます！
```suggestion
test("アクセント分割したらアクセント区間が増える", async ({ page }) => {
```